### DISCUSSION TITLE
CVE-2023-24163 请问该漏洞修复计划？

### DISCUSSION THREAD

**Discussion by Tinsleyzh** [looly/hutool#2884] [Q&A]
https://nvd.nist.gov/vuln/detail/CVE-2023-24163

**Comment by Tinsleyzh** [comment]
还有这个漏洞：CVE-2023-24162

**Reply by looly** [reply]
https://gitee.com/dromara/hutool/issues/I6AEX2

5.8.12会删掉这个方法。

**Comment by looly** [comment]
> https://nvd.nist.gov/vuln/detail/CVE-2023-24163

这个不是hutool本身的漏洞，而是脚本引擎的问题。

**Reply by Tinsleyzh** [reply]
1、脚本引擎是指哪个脚本引擎呢？
2、这个漏洞该怎么处理，等脚本引擎修复吗？

**Reply by looly** [reply]
脚本引擎是，JDK8默认是js脚本，你可以通过引入jar使用grovvy、python等脚本，那这块的漏洞处理就不可能。
